src/main/java/com/qaprosoft/carina/demo/mobile/gui/pages/common/ContactUsPageBase.java

```diff
@@ -20,5 +22,10 @@ public ContactUsPageBase(WebDriver driver) {
     public abstract boolean isSuccessMessagePresent();
 
     public abstract boolean isRecaptchaPresent();
-
+    
+    public void me() {
+        for (int i=0; i<10; i++) {
+            System.out.println(i);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://demo.qaprosoft.com/sonarqube/coding_rules#rule_key=squid%3AS106)